Assets/MixedRealityToolkit.Tests/EditModeTests/Core/TestFixture_01_MixedRealityToolkitTests.cs

```diff
@@ -594,10 +595,90 @@ public void Test_07_02_DisableServicesByType()
             }
         }
 
+        #region Multiple Instances Tests
+
+        [Test]
+        public void Test_08_01_CreateMultipleInstances()
```

Comment:
Per @wiwei suggestion I think we are trying to move away from numberint tests, just use the test name without numbers.

Reply:
Do you think we should make that change in this PR, or in a future PR?